src/main/java/com/sktTask/ProductService/Receiver.java

```diff
@@ -0,0 +1,23 @@
+package com.sktTask.ProductService;
+
+import java.util.concurrent.CountDownLatch;
+
+import com.sktTask.ProductService.models.Product;
+import org.apache.logging.log4j.message.Message;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Service;
+
+@Service
+public class Receiver {
+
+    private static final Logger log = LoggerFactory.getLogger(Receiver.class);
+
+    @RabbitListener(queues = ProductServiceApplication.QUEUE_NAME)
```

Comment:
use `application.yml` files, and much better if you use this https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-typesafe-configuration-properties (check advantages and disadvantages)